tron/core/actionrun.py

```diff
@@ -646,9 +646,12 @@ def remaining_triggers(self):
         ]
 
     def success(self):
-        if self.trigger_downstreams:
-            self.emit_triggers()
-        return self._done('success')
+        transitionvalid = self._done('success')
+        if transitionvalid:
+            if self.trigger_downstreams:
+                self.emit_triggers()
+
+        return transitionvalid
```

Comment:
super-duper-nit: we should use `snake_case` for variables
```suggestion
        transition_valid = self._done('success')
        if transition_valid:
            if self.trigger_downstreams:
                self.emit_triggers()

        return transition_valid
```

Reply:
Done 👍